canonical-websites/cn.ubuntu.com
# Add 1704 takeover

## Done

Add 17.04 takeover

## QA

- Follow the instructions in README.md to get started
- Go to http://localhost:8001/
- Check the page in s/m and l, check the code and check against copy doc

Doc: https://docs.google.com/document/d/1r2v58SP7xJdNSasB-G_NNGqIFi42XRZeLmDK0LVmSSA/edit#
